pkg/kando/location.go

```diff
@@ -38,17 +46,85 @@ func newLocationCommand() *cobra.Command {
 	cmd.AddCommand(newLocationDeleteCommand())
 	cmd.PersistentFlags().StringP(pathFlagName, "s", "", "Specify a path suffix (optional)")
 	cmd.PersistentFlags().StringP(profileFlagName, "p", "", "Pass a Profile as a JSON string (required)")
-	_ = cmd.MarkFlagRequired(profileFlagName)
+	cmd.PersistentFlags().StringP(repositoryServerFlagName, "r", "", "Pass a Repository Server CR as a JSON string (required for kopia based blueprints)")
 	return cmd
 }
 
+// NewDataMover creates an instance of DataMover Interface and returns
+// the preferred DataMover as per the arguments passed in kando command
+func NewDataMover(dm *DataMoverVal, outputName, snapJson string) datamover.DataMover {
+	switch dm.Type {
+	case profileFlagName:
+		return &datamover.Profile{
+			OutputName: outputName,
+			Profile:    dm.Profile,
+			SnapJSON:   snapJson,
+		}
+	case repositoryServerFlagName:
+		return &datamover.RepositoryServer{
+			OutputName:       outputName,
+			RepositoryServer: dm.RepositoryServer,
+			SnapJSON:         snapJson,
+		}
+	default:
```

Comment:
```suggestion
	default:
```
I think we can remove this, and return nil directly at the end

Reply:
Yes, I think we could. I'm keeping it in case. Because in many places we're having default case.